### ISSUE TITLE
Markers in tree view

### ISSUE THREAD

**Issue by averrin** [paulpflug/color-tabs#14]
Maybe you package would be provide something like this?

![tree-view-marker](https://cloud.githubusercontent.com/assets/426007/8647956/5b50d766-2965-11e5-80d0-14a6eb106fbc.png)

**Comment by paulpflug** [comment]
I think this is out of the scope of this package
But it could be easily be done in a separate package? Seems easier than coloring tabs ;)
Maybe a package just providing a service to add a marker to a path?

**Comment by averrin** [comment]
Yeah, its slightly out of scope, but 90% of new code already exists in this package.
